src/chrome/content/rules/Tum.de.xml

```diff
@@ -1,10 +0,0 @@
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://tum.de/ => https://www.tum.de/: (6, 'Could not resolve host: tum.de')
--->
-<ruleset name="Tum.de" default_off='failed ruleset test'>
-  <target host="tum.de" />
```

Comment:
You should give a hint in the comments on the top of Technical_University_Munich.xml  that ^tum.de and ^tu-muenchen.de do not exist.